richardaspinall/poker-react-node
# feat: create account and sign in forms

### Description


This PR adds account creation and signing in forms, they currently don't work (awaiting backend support) 

Note: they majorly duplicate each other and use the same `scss` file (duplicating the styling) which for now is going to be on purpose until a later possible refactor and as we figure out global styling.

### How to test



1. Start server `npm run start:dev` 
2. Head to the Home page
3. Head to the `/createaccount` and `/signin` routes 

Note: they both don't work, `/createaccount` should return a `method_not_implemented` error and `/signin` should fail)
<img width="941" alt="Screenshot 2024-03-26 at 8 21 16 am" src="https://github.com/richardaspinall/poker-react-node/assets/15721687/7d6f7b3a-fc5d-4064-b507-d2faacf9e982">

### Task

[CU-86cuu33v9](https://app.clickup.com/t/86cuu33v9) 

### Checklist
